rclcpp/QUALITY_DECLARATION.md

```diff
@@ -137,20 +137,63 @@ Currently nightly test results can be seen here:
 
 ## Dependencies [5]
 
+Below are evaluations of each of `rclcpp`'s run-time and build-time dependencies that have been determined to influence the quality.
+
+It has several "buildtool" dependencies, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
+It also has several test dependencies, which do not affect the resulting quality of the package, because they are only used to build and run the test code.
+
+
 ### Direct and Optional Runtime ROS Dependencies [5.i]/[5.ii]
 
 `rclcpp` has the following runtime ROS dependencies:
-  - libstatistics_collector
-  - rcl
-  - rcl_yaml_param_parser
-  - rcpputils
-  - rcutils
-  - rmw
-  - statistics_msgs
-  - tracetools
 
-It has several "buildtool" dependencies, which do not affect the resulting quality of the package, because they do not contribute to the public library API.
-It also has several test dependencies, which do not affect the resulting quality of the package, because they are only used to build and run the test code.
+#### `libstatistics_collector`
+
+The `libstatistics_collector` package provides a lightweight aggregation utilities to collect statistics and measure message metrics.
+
+It is **Quality Level 4**, see its [Quality Declaration document](https://github.com/ros-tooling/libstatistics_collector/tree/master/Quality_Declaration.md).
+
+#### `rcl`
+
+`rcl` is the ROS 2 client library in C.
```

Comment:
I think `rcl` is not intented to be a client library, but a core used for the client libraries. I'd use a similar text used in its README:
```suggestion
`rcl` is a library to support implementation of language specific ROS 2 Client Libraries..
```